contracts/BSCValidatorSet.sol

```diff
@@ -90,14 +97,18 @@ contract BSCValidatorSet is IBSCValidatorSet, System, IParamSubscriber, IApplica
     uint256 enterMaintenanceHeight;     // the height from where the validator enters Maintenance
     bool isMaintaining;
 
+    // BEP-126 Fast Finality
+    bytes voteAddress;
+
     // reserve for future use
-    uint256[20] slots;
+    uint256[19] slots;
```

Comment:
What is the reason for this change?

Reply:
Because we add a new field, so the placeholder should be reduced by one